src/attributes/classes.js

```diff
@@ -86,7 +84,7 @@ jQuery.fn.extend({
 					}
 
 					// Only assign if different to avoid unneeded rendering.
-					finalValue = value ? jQuery.trim( cur ) : "";
```

Comment:
Here, it was for removing all the classes when we call `.removeClass()` without any argument, but I write a special `if` (with the `.attr`) only for this case.